src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -745,7 +748,7 @@ class CreateVmModal extends React.Component {
         }
         super(props);
         this.state = {
-            inProgress: false,
+            inProgress: INVALID,
```

Comment:
As the other comment was resolved: `inProgress` sounds boolean-y. Please rename to e.g. `createMode`, or similar.